hammer_cli_csv.gemspec

```diff
@@ -6,10 +6,16 @@ Gem::Specification.new do |spec|
 
   spec.name = "hammer_cli_csv"
   spec.version = HammerCLICsv.version
-  spec.authors = ["Thomas McKay"]
-  spec.email = ["thomasmckay@redhat.com"]
+  spec.authors = [
+    "Thomas McKay",
+    "Eric Helms",
+    "Adam Price",
+    "Andrew Kofink",
+    "David Davis"
+  ]
+  spec.email = ["katello@lists.fedorahosted.org"]
```

Comment:
I used that email because that's what [hammer-cli-katello uses](https://github.com/Katello/hammer-cli-katello/blob/master/hammer_cli_katello.gemspec#L36)